src/metabase/task/send_pulses.clj

```diff
@@ -35,6 +35,11 @@
                                      u.cron/schedule-map->cron-string
                                      (str/replace " " "_")))))
 
+(defn- send-pulse-trigger-key->schedule-map
```

Comment:
🤪

Reply:
this returns a cron-string, not a schedule map. Should the name be 
`send-pulse-trigger-key->cron-string`?